### DISCUSSION TITLE
The Status of the DT Package | /en/2023/05/dt-status/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1488] [Comments]
My first job at RStudio (now rebranded to Posit) was to integrate the JS library DataTables into Shiny. That was in 2013, almost ten years ago. To be honest, I have never been highly interested in &hellip;

https://yihui.org/en/2023/05/dt-status/

**Comment by DTsai2** [comment]
Hi YiHui,
 A random beginner of R who use DT package to build tables for my company here. Thank you for this decent and great package. I use DT in a lot of scenarios such as Shiny and RMarkdown report. Your contribution really make me and my colleague work and solve problem effectively!

謝謝你!

**Reply by yihui** [reply]
Thank you for the kind words!
